source/frontend/src/features/properties/map/acquisition/common/AcquisitionHeader.test.tsx

```diff
@@ -48,4 +48,24 @@ describe('AcquisitionHeader component', () => {
     expect(getByText('File:')).toBeVisible();
     expect(getByText('1-12345-01 - Test ACQ File')).toBeVisible();
   });
+
+  it('renders the file Project Number and name concatenated', async () => {
+    const testAcquisitionFile = mockAcquisitionFileResponse();
+    const { getByText } = setup({ acquisitionFile: testAcquisitionFile });
+
+    expect(getByText('Ministry project:')).toBeVisible();
+    expect(
+      getByText(
+        "001 - Hwy 14 Expansion - Vancouver Island but it's really long so it can wrap around if it has to",
+      ),
+    ).toBeVisible();
+  });
+
+  it('renders the file Product code and description concatenated', async () => {
+    const testAcquisitionFile = mockAcquisitionFileResponse();
+    const { getByText } = setup({ acquisitionFile: testAcquisitionFile });
+
+    expect(getByText('Ministry product:')).toBeVisible();
+    expect(getByText('00048 - MISCELLANEOUS CLAIMS')).toBeVisible();
+  });
```

Comment:
Please add one where the code/description is null or undefined given there is some logic there to filter those out

Reply:
Should I still need to make that test since both fields are not nullable in the backend?

Reply:
The testing code should not make any assumptions that are not explicit in the code being tested. In this case, the codes can be undefined or null according to the types used in the model.

Also, we do not use explicit nulls on the backend except for numeral primitives. That way any object reference could be null

Reply:
Updated

Reply:
Thank you for updating. I should have been more explicit; The code that does the format is what needs testing given it has some logic around it[1].
So a acq with a product, but with either code or description being null

[1]``
 {formatMinistryProduct(
                acquisitionFile?.product?.code,
                acquisitionFile?.product?.description,
              )}
``

Reply:
Removed unnecessary logic.